docs/log-management/index.md

```diff
@@ -174,17 +163,15 @@ search_settings:
 ## UI Integration
 
 Currently, Quickwit provides a simplistic UI to get basic information from the cluster, indexes and search documents.
-If a simple UI is not sufficient for you but you need additional features, Grafana and Elasticsearch query API support are planned for Q2 2023, stay tuned!
-
-Please open an issue on [GitHub](https://github.com/quickwit-oss/quickwit) if you want to have other integrations.
+If a simple UI is not sufficient for you and you need additional features, Grafana and Elasticsearch query API support are planned for Q2 2023, stay tuned!
```

Comment:
Is the jaeger support not something which may be interesting to put here?

Reply:
Not for logs. I have a separated PR for distributed tracing here: https://github.com/quickwit-oss/quickwit/pull/2947
But not yet ready for review.

Reply:
Hmm maybe worth associating the two pages then, however, if I'm reading through everything it may pique my interest to move on to the distributed tracing stuff and vice versa.

Reply:
Good point indeed.